compiler/rustc_ast/src/ast.rs

```diff
@@ -2979,6 +2980,10 @@ pub struct AttrItem {
     pub args: AttrArgs,
     // Tokens for the meta item, e.g. just the `foo` within `#[foo]` or `#![foo]`.
     pub tokens: Option<LazyAttrTokenStream>,
+    /// The span of the contents of the attribute.
```

Comment:
Can this not be computed on the fly?

Reply:
Like, adding a `fn span` to `AttrItem` that joins the span of the path to the `args` (well, matching on what it is, and joining that span if it's nonempty?)

Reply:
Oh, lmao